src/tasks/mod.rs

```diff
@@ -1,5 +1,6 @@
 pub mod block;
 pub mod bundler;
 pub mod oauth;
+pub mod receipts;
```

Comment:
super nit: this mod name is plural, but the rest are in the singular.

Reply:
i ended up renaming the whole thing, i think makes better sense to call it `metrics`, and i kept it plural because `metric` sounds weird https://github.com/init4tech/builder/pull/48/commits/7ebd8d219556a9fa66d2c3c697b445c254d6029c